### DISCUSSION TITLE
Provide minimal sample

### DISCUSSION THREAD

**Discussion by ephb** [loh0123/LohFunctionPlugin#2] [Ideas]
Hello,
I am trying to understand your project but I am unsure how to set up the scene for it to render anything.
Would it be possible to provide a sample scene, blueprint or some documentation?

**Comment by loh0123** [comment]
https://github.com/loh0123/LFPExample

Might not be the best example but it will show you how to setup Voxel

**Reply by ephb** [reply]
Awesome, thank you. I will check it out right away.

**Reply by ephb** [reply]
That was very helpful. I would have never got it working by myself. Quite the complex setup :)

I did try to reach out via email to you. Is there any chance you are open for contract work?

**Reply by loh0123** [reply]
Thank you, but I'm currently not free with work and all the stuff going on in my life, I will still continue to update the plugin slowly
